src/Pages/Signup/Signup.js

```diff
@@ -1 +1,152 @@
-export default () => <h1>signup</h1>;
+import React from 'react';
+import styled from 'styled-components/macro';
+import { flexCenter } from '../../style/mixin';
+import { Link } from 'react-router-dom';
+import LoginLinkWrapper from '../../Components/LoginLinkWrapper';
+
+export default function Signup() {
+  return (
+    <>
+      <SignupContainer>
+        <HelloImage src="http://localhost:3000/images/waving-hand.png" />
+        <Title>반갑습니다!</Title>
+        <Paragraph>여행의 모든 것, 마이위시트립</Paragraph>
+        <Button>
+          <KakaoIcon src="http://localhost:3000/images/kakao.svg" />
+          카카오로 계속하기
+        </Button>
+        <LoginLinkContainer>
+          {LINK_DATA &&
+            LINK_DATA.map((link, index) => {
+              return (
+                <>
+                  <LoginLinkWrapper
+                    name={link.name}
+                    url={link.url}
+                    icon={link.icon}
+                    iconGray={link.iconGray}
+                  ></LoginLinkWrapper>
```

Comment:
```suggestion
                  <LoginLinkWrapper
                   link={link}
                  / >
```
이런식으로 link 전체를 보내주고, LoginLonkWrapper 에서 받아서 쓸 때 구조분해 할당을 해서 사용하면 더 간단하겠네요 :)

Reply:
셀프클로징으로도 바꿨습니다!